package.json

```diff
@@ -45,59 +47,86 @@
     "@types/qunit": "^2.5.3",
     "@types/rsvp": "^4.0.3",
     "babel-eslint": "^10.0.2",
+    "babel-plugin-debug-macros": "^0.3.2",
```

Comment:
a lot more things were added to this file than I would have thought, I feel many likely should have been placed into the test-infra package?

Reply:
I hoisted all dependencies to the main package.json file, and synchronized versions. Putting them here as well as in the individual package.json files in each package ensures that the same version is used across all packages.

The other reason I did this is I got warnings running https://github.com/mybigday/yarn-workspace-sync-deps

Reply:
there were a few things that needed fixed (I noticed a few dangling dev-deps with "*") but there's already a `sync` command within the repo that does what we want. It looks like this is hoisting `dependencies` in addition to `devDependencies` which is not something I think we want at this point.

Reply:
I got errors running that script.